runner/config/test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "safelist": ["b4bf0fcd-1dd3-4650-92fe-d1f83885a447.mock.pstmn.io"]
```

Comment:
If this is just a mock URL lets make it something that will never be a real one, like xyz.example.org, or a cyb.dev subdomain. If it's used in the tests... will it always be available?